include/itkTileMontage.hxx

```diff
@@ -24,8 +24,12 @@
 #include "itkMultiThreaderBase.h"
 #include "itkNumericTraits.h"
 
+#include "itk_eigen.h"
+#include ITK_EIGEN( Sparse )
```

Comment:
Just curious -- what does this do?

Reply:
Selects either internal Eigen's header, or system Eigen's header, based on CMake configuration. I copy-pasted this snippet from SymmetricEigenAnalysis class which @phcerdan modified after adding Eigen to ITK.